arviz/plots/plot_utils.py

```diff
@@ -42,31 +42,42 @@ def _scale_fig_size(figsize, textsize, rows=1, cols=1):
     markersize : int
         markersize
     """
-    rc_width, rc_height = tuple(mpl.rc_params()['figure.figsize'])
-    rc_fontsize = mpl.rc_params()['font.size']
-    rc_linewidth = mpl.rc_params()['lines.linewidth']
-    rc_markersize = mpl.rc_params()['lines.markersize']
+    params = mpl.rcParams
+    rc_width, rc_height = tuple(params['figure.figsize'])
+    rc_ax_labelsize = params['axes.labelsize']
+    rc_titlesize = params['axes.titlesize']
+    rc_xt_labelsize = params['xtick.labelsize']
+    rc_linewidth = params['lines.linewidth']
+    rc_markersize = params['lines.markersize']
+    if isinstance(rc_ax_labelsize, str):
+        rc_ax_labelsize = 15
+    if isinstance(rc_titlesize, str):
+        rc_titlesize = 16
+    if isinstance(rc_xt_labelsize, str):
+        rc_xt_labelsize = 14
+
     if figsize is None:
         width, height = rc_width, rc_height
-        width = width * cols
-        height = height * rows
+        sff = 1 if (rows == cols == 1) else 1.15
```

Comment:
How are these magic numbers decided?

Reply:
I was trying to make that for a figure with multiple subplots each subplot has the same size as a single figure.

Reply:
Got it!